core/src/apps/management/change_pin.py

```diff
@@ -28,7 +28,11 @@ async def change_pin(ctx: Context, msg: ChangePin) -> Success:
     await _require_confirm_change_pin(ctx, msg)
 
     # get old pin
-    curpin, salt = await request_pin_and_sd_salt(ctx, "Enter PIN")
+    if msg.remove:
+        prompt = "Enter PIN"
+    else:
+        prompt = "Enter old PIN"
```

Comment:
This longer text is colliding with `15 attempts left` - cannot find the UI screenshots now

Reply:
![image](https://user-images.githubusercontent.com/42543243/236181682-427785dd-fa92-4c9e-a6ab-3aff483b23b1.png)

I guess I will remove the `old`, so it is as before

Reply:
Can [this](https://github.com/trezor/trezor-firmware/issues/2996) be done together with this PR, or is it something else?

Reply:
> Can https://github.com/trezor/trezor-firmware/issues/2996 be done together with this PR, or is it something else?

I would do it in a separate one